ironmon_tracker/Program.lua

```diff
@@ -1,5 +1,7 @@
 Program = {
 	currentScreen = 1,
+	inStartMenu = false,
+	startMenuCallback = 0,
```

Comment:
Oh just noticed i forgot to remove this lol

was trying out reading the callback function in the start menu at one point but it didn't work nicely so removed it but accidentally left this variable in

Not anything breaking since it's not doing anything but this `startMenuCallback` can be safely deleted somepoint